pkg/storaged/manifest.json

```diff
@@ -56,7 +56,11 @@
             "opensuse": "nfs-client", "opensuse-leap": "nfs-client",
             "debian": "nfs-common", "ubuntu": "nfs-common"
         },
-        "vdo_package": { "rhel": "vdo", "centos": "vdo" }
+        "vdo_package": { "rhel": "vdo", "centos": "vdo" },
+        "stratis_package": { "fedora": "stratis-cli",
```

Comment:
We need both, but we can only install a single package here. Possible alternative, replace "stratis key set" with our own little Python program.  The only reason we can't use the D-Bus call is because it expects a file descriptor to be passed in.

Reply:
Done.